ICGE-Simulation/src/main/java/de/unistuttgart/informatik/fius/icge/simulation/inspection/InspectionData.java

```
@@ -59,8 +59,10 @@ public Object getAttributeValue(final Object obj, final String name) {
         if (p != null) {
             try {
                 return p.getValue(obj);
-            } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
+            } catch (IllegalAccessException | IllegalArgumentException e) {
+                // TODO Auto-generated catch block
                 e.printStackTrace();
+            } catch (InvocationTargetException e) {
```

Add a comment that explains why the exception is just silently suppressed.